src/main/java/core/basesyntax/dao/writeReportImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.model.Account;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class writeReportImpl implements writeReport {
+    Account account = new Account();
```

Comment:
The `Account` instance is created within this class, which might not reflect the actual state of accounts if it's not shared or updated correctly. Consider passing an `Account` instance to this class or method to ensure it uses the correct data.